realm/realm-library/src/main/java/io/realm/internal/Table.java

```diff
@@ -30,13 +30,14 @@
  */
 public class Table implements TableSchema, NativeObject {
 
-    public static final int TABLE_MAX_LENGTH = 56; // Max length of class names without prefix
+    private static final String TABLE_PREFIX = Util.getTablePrefix();
+    public static final int TABLE_NAME_MAX_LENGTH = 62; // Max length of table names
```

Comment:
https://github.com/realm/realm-core/blob/655595ac410fd98318c9e84ad5b7a75ac88232ab/src/realm/group.hpp#L328 should be `63` i think?
`StringData::size()` doesn't count the `\0`. 

https://realm.io/docs/objc/latest/#current-limitations
> Class names are limited to a maximum of 57 UTF8 characters.